backend/src/upload/views.py

```diff
@@ -104,19 +102,19 @@ async def get_history(user_id: int) -> HistoryResponseModel:
             "id": 1,
             "index": 1,
             "filename": "article about machine learning",
-            "status": UploadStatus.READY,
+            "status": CeleryTaskStatus.SUCCESS,
         },
         {
             "id": 5,
             "index": 2,
             "filename": "article about cloud computing",
-            "status": UploadStatus.PENDING,
+            "status": CeleryTaskStatus.PENDING,
         },
         {
             "id": 3,
             "index": 3,
             "filename": "article about something else",
-            "status": UploadStatus.ERROR,
+            "status": CeleryTaskStatus.FAILURE,
         },
     ]
 
```

Comment:
This wont work on frontend

Reply:
I know, but this is what we need to adjust.

Reply:
https://github.com/pwr-TeamProject23/M2/blob/dev/frontend/src/fileUpload/components/models.ts